opm/autodiff/AQUCT_params.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef OPM_AQUCT_HEADER_INCLUDED
```

Comment:
the copyright header is missing here. also, please to avoid underscores in filenames, i.e. rename that file to `AquctParams.cpp`.

Reply:
@andlaus Thanks for the comment, but this file is no longer in use as it was a temporary fix while I wait for the official opm-parser aquifer object. It is now done and removed.

Reply:
the file is still part of the PR, right? If you don't need it could you please remove it?

Reply:
It's removed already. Thanks for the reminder. :-)